Alignment/OfflineValidation/python/TkAlAllInOneTool/offlineValidationTemplates.py

```diff
@@ -33,7 +33,7 @@
  ## Load and Configure OfflineValidation and Output File
  ##
 process.load("Alignment.OfflineValidation.TrackerOfflineValidation_.oO[offlineValidationMode]Oo._cff")
-process.TrackerOfflineValidation.oO[offlineValidationMode]Oo..compressionSettings = compressionSettings,
+process.TrackerOfflineValidation.oO[offlineValidationMode]Oo..compressionSettings = compressionSettings
```

Comment:
@TomasKello please remove this, as it has already been taken care of by PR https://github.com/cms-sw/cmssw/pull/34876